server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -448,6 +450,34 @@ public void partialUpdateSecret_notFound() throws Exception {
   }
 
 
+  //---------------------------------------------------------------------------------------
+  // secretContentsAtVersion
+  //---------------------------------------------------------------------------------------
+
+  @Test public void secretContentsAtVersion_success() throws Exception {
```

Comment:
Wdyt about a test case that checks that `Optional.empty()` is returned when the secret/version isn't found?

Reply:
Can't hurt - I'll add one real quick